bin/Download_scripts/topaz_get_mercator_bio_CMT.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
I think all of this should be inside "user specification as it will change depending on which machine you are working on.

Reply:
done